### ISSUE TITLE
VSCode Insiders - issue with Salesforce / LWC / HTML

### ISSUE THREAD

**Issue by simonhyphen8** [microsoft/vscode#146703]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.67.0 Insider
- OS Version: Mac OS Monterey 12.2.1

Steps to Reproduce:

1. Open Salesforce Lightning Web Component - HTML File
2. Move mouse pointer over any element in the HTML
3. Error message below:

[Error - 7:53:32 AM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read properties of undefined (reading 'forEach')
  Code: -32603

**Comment by gjsjohnmurray** [comment]
Maybe caused by a Salesforce extension.

Please use the Start Extension Bisect command to investigate.

**Comment by gjsjohnmurray** [comment]
1. Open Command Palette (press F1, or use context menu from gearwheel at bottom of Activity Bar).
2. Start typing the command.

**Comment by simonhyphen8** [comment]
Thanks, I've not used that command before.
I received this report back:
Extension Bisect is done and has identified salesforce.salesforcedx-vscode-lwc as the extension causing the problem.
Is this something I need to report to Salesforce?

**Comment by gjsjohnmurray** [comment]
Yes. You may be able to open the report via "Report Issue" on the "Help" menu. Choose "An extension" in the "File on" dropdown. Then pick the correct extension.

/causedByExtension
